tools/buildpkg.sh

```diff
@@ -122,21 +119,22 @@ git submodule -q update --init --recursive >/dev/null || exit 1
 echo "Done"
 
 echo "Installing composer dependencies:"
+curl -sS https://getcomposer.org/installer | php
 echo -n " - lib/pkp ... "
 cd lib/pkp
-composer.phar update --no-dev
+php ../../composer.phar update --no-dev
 cd ../..
 echo "Done"
 
 echo -n " - plugins/paymethod/paypal ... "
 cd plugins/paymethod/paypal
-composer.phar install --no-dev
+php ../../../composer.phar install --no-dev
```

Comment:
Do we normally keep a copy of  `composer.phar` in the root of the project? @asmecher talked about using it from path, but I've always just used it as an executable: `composer install`.

Reply:
I believe that now, that we have several directories/paths it is ok to have composer.phar in the root of the project -- it is the same in the README :-)
But, when you just use composer install, do you need composer.php?

Reply:
@asmecher we still need your input on how to reference composer. See the comments above.